fontforge/cvexport.c

```diff
@@ -726,10 +733,10 @@ static void MakeExportName(char *buffer, int blen,char *format_spec,
 		for ( pt=unicode; *pt!='\0' && buffer<bend; )
 		    *buffer++ = *pt++;
 	    } else if ( ch=='e' ) {
-		sprintf( unicode,"%d", (int) map->backmap[sc->orig_pos] );
-		for ( pt=unicode; *pt!='\0' && buffer<bend; )
-		    *buffer++ = *pt++;
-	    } else
+        sprintf( unicode,"%d", (int) map->backmap[sc->orig_pos] );
```

Comment:
We try not to reformat blocks that aren't being changed otherwise so that `git blame` gives a more usefully direct answer.